css/80_app.css

```diff
@@ -2198,9 +2198,55 @@ div.combobox {
     width: 100%;
     overflow: hidden;
 }
+
+.adv-opt-reference {
+    flex: 1 1 auto;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    overflow: hidden;
+    white-space: normal !important;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+    padding: 0px 5px 0px 5px;
+}
+
+.adv-top {
+    margin-top: 10px;
+}
+
+.adv-opt-icon {
+    opacity: .5;
+}
+
+.hoot-field-label-button {
+        border: 1px solid #ccc;
```

Comment:
I think it looks fine without the border. adding the border creates the appearance of a double :rainbow:, oops i mean border!